src/me/mrCookieSlime/Slimefun/Lists/SlimefunItems.java

```diff
@@ -376,7 +376,7 @@
 	public static ItemStack DIGITAL_MINER = new CustomItem(Material.IRON_PICKAXE, "&bDigital Miner", "", "&a&oDigs out everything!");
 	public static ItemStack ADVANCED_DIGITAL_MINER = new CustomItem(Material.DIAMOND_PICKAXE, "&6Advanced Digital Miner", "", "&a&oDigs out everything!", "&a&oAutomatically crushes your Ores");
 	public static ItemStack AUTOMATED_PANNING_MACHINE = new CustomItem(Material.BOWL, "&aAutomated Panning Machine", "", "&a&oA MultiBlock Version of the Gold Pan");
-
+	public static ItemStack OUTPUT_CHEST = new CustomItem(Material.CHEST, "&4Output Chest", "", "&c&oA basic machine will try to put", "&c&oitems in this chest if it's placed", "&c&oadjacent to the dispenser.");
```

Comment:
This description sounds a little misleading as the chest itself doesn't put the item into itself.
I'd keep it much more simple.

Suggestion:
"Place this Chest next to a basic Machine and any crafted Items will go into this Chest instead"
(Rough, feel free to improve it)

Reply:
I don't really understand the confusion here.

It says "A basic machine", this is not the same as referring to "THE basic machine". If anything, it refers to anything BUT the output chest. I urge you to reconsider, and read through the sentence once more.

"A basic machine will try to put items in this chest if it's placed adjacent to the dispenser"

Reply:
Wow, my bad.
Sorry, I have no idea what I read there the first time.